data/chamberezigbo.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "Chamberlain Ezigbo",
   "type": "personal",
-  "bio": "JavaScript Engineer | Front-end Developer",
-
+  "bio": "JavaScript Engineer | Full-Stack Developer",
+  "avatar": "https://github.com/chamberezigbo.png",
```

Comment:
As of our latest update, we no longer need the `avatar` key, so please remove this one. 

```suggestion
```